cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -52,7 +52,10 @@ def self.config(env)
 
     # Signed-in user type (student/teacher), or signed-out if cookie is not present.
     user_type = "_user_type#{env_suffix}"
-    default_cookies = DEFAULT_COOKIES + [user_type]
+    # Students younger than 13 shouldn't see App Lab and Game Lab unless they
+    # are in a teacher's section for privacy reasons.
+    limit_project_types = "_limit_project_types#{env_suffix}"
+    default_cookies = DEFAULT_COOKIES + [user_type] + [limit_project_types]
```

Comment:
Nit: Since you're adding arrays, these don't need to be separate:
```rb
default_cookies = DEFAULT_COOKIES + [user_type, limit_project_types]
```

Reply:
Bigger question: do I even need to do this?  I was just following the `user_type` cookie example.